### PR TITLE
add filename to results side panel

### PR DESCRIPTION
**Pull request by jamesgorrie**
The filename is busy being added to the caption so they can do this at the moment (see the filename and results at the same time).

Seems to make sense to just put it there.

### CODE REVIEW

File: kahuna/public/js/services/image-accessor.js [hunk 1]
```diff
@@ -29,6 +29,12 @@ imageAccessor.factory('imageAccessor', function() {
         return image.data.metadata;
     }
 
+    function readExtraInfo(image) {
+        return {
+            filename: image.data.uploadInfo && image.data.uploadInfo.filename
```
Comment (jamesgorrie) [inline kahuna/public/js/services/image-accessor.js]: I've gone with `extraInfo` as we might want something else in there at some stage.

**Comment by theefer** [comment]
:+1:

**Comment by tsop14** [comment]
Given how similar they are, would it make sense to have all the info on the image page in this search side panel?

**Comment by theefer** [comment]
> Given how similar they are, would it make sense to have all the info on the image page in this search side panel?

+1

**Comment by jamesgorrie** [comment]
@tsop14 :+1:

**Pull request merged**